File: javascript/1005/main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

function averageOne() {
  var a = parseFloat(lines[0]);
  var b = parseFloat(lines[1]);

  var weightOfA = 3.5;
  var weightOfB = 7.5;

  var media = (a * weightOfA + b * weightOfB) / (weightOfA + weightOfB);

  return `MEDIA = ${media.toFixed(5)}`;
}

console.log(averageOne());

File: python/1005/main.py
def average_one():
    a = float(input())
    b = float(input())
    weight_of_a = 3.5
    weight_of_b = 7.5
    media = (a * weight_of_a + b * weight_of_b) / (weight_of_a + weight_of_b)
    return f'MEDIA = {media:.5f}'


if __name__ == '__main__':
    print(average_one())
